src/components/app-content-renderer/styles/panels.scss

```diff
@@ -145,8 +153,11 @@
     }
 
     .recommendation-button {
-      width: 100%;
-      height: fit-content;
+      width: 100px;
+      height: 30px;
```

Comment:
Do we need the `height` prop here?

Reply:
@karelhala Yes, we do for the moment. It's helping prevent the buttons split between columns - it's a WIP